source/common/request_stream_grpc_client_impl.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <queue>
+#include <string>
+
+#include "envoy/grpc/async_client.h"
+#include "envoy/grpc/async_client_manager.h"
+#include "envoy/stats/scope.h"
+#include "envoy/upstream/cluster_manager.h"
+
+#include "nighthawk/common/request.h"
+#include "nighthawk/common/request_stream_grpc_client.h"
+
+#include "external/envoy/source/common/common/logger.h"
+#include "external/envoy/source/common/grpc/typed_async_client.h"
+#include "external/envoy/source/common/http/header_map_impl.h"
+
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic warning "-Wunused-parameter"
+#endif
+#include "api/request_source/service.grpc.pb.h"
+
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
+
+namespace Nighthawk {
+
+class ProtoRequestHelper {
+public:
+  static RequestPtr
+  messageToRequest(const Envoy::Http::HeaderMap& base_header,
+                   const nighthawk::request_source::RequestStreamResponse& message);
+};
+
+class RequestStreamGrpcClientImpl
```

Comment:
Similarly here, can we add a short comment explaining what the class is for and document its constructor?